pkg/compliance/clustercompliancereport_test.go

```diff
@@ -216,3 +216,28 @@ func getDetailReport(ctx context.Context, namespaceName types.NamespacedName, cl
 	}
 	return &report, nil
 }
+
+func getStarboardConfig() (starboard.ConfigData, error) {
+	clientset := fakesimple.NewSimpleClientset(
+		&corev1.ConfigMap{
+			ObjectMeta: metav1.ObjectMeta{
+				Namespace: starboard.NamespaceName,
+				Name:      starboard.ConfigMapName,
+			},
+			Data: map[string]string{
+				"compliance.failEntriesLimit": "1",
+			},
+		},
+		&corev1.Secret{
+			ObjectMeta: metav1.ObjectMeta{
+				Namespace: starboard.NamespaceName,
+				Name:      starboard.SecretName,
+			},
+			Data: map[string][]byte{
+				"baz": []byte("s3cret"),
```

Comment:
Is this secret used somehow in the test scenarios? If it's not required there is no point in adding it to test fixture.